packages/backend-app-api/package.json

```diff
@@ -50,7 +50,7 @@
     "test": "backstage-cli package test"
   },
   "dependencies": {
-    "@backstage/backend-common": "workspace:^",
+    "@backstage/backend-common": "^0.24.1",
```

Comment:
@freben should it be `[0.25.0-next.0](https://www.npmjs.com/package/@backstage/backend-common/v/0.25.0-next.0)` instead?

Reply:
Excellent callout - there were actually some interesting things in the latest next line release. Let's hold off on this until next mainline. Marked as draft and added the merge-after-release label

Reply:
These are now all updated to `^0.25.0` after the release